qiskit/transpiler/passes/synthesis/high_level_synthesis.py

```diff
@@ -720,11 +756,43 @@ class KMSSynthesisLinearFunction(HighLevelSynthesisPlugin):
 
     This plugin name is :``linear_function.kms`` which can be used as the key on
     an :class:`~.HLSConfig` object to use this method with :class:`~.HighLevelSynthesis`.
+
+    The plugin supports the following plugin-specific options:
+
+    * use_inverted: Indicates whether to run the algorithm on the inverse matrix
+        and to invert the synthesized circuit.
+        In certain cases this provides a better decomposition then the direct approach.
```

Comment:
then --> than

Reply:
thanks for catching this, would be fixed by https://github.com/Qiskit/qiskit/pull/12316